tap_zendesk/client.py

```diff
@@ -146,5 +265,44 @@ def post_process(
         Returns:
             The updated record dictionary, or ``None`` to skip the record.
         """
-        # TODO: Delete this method if not needed.
         return row
+
+
+class IncrementalZendeskStream(ZendeskStream):
+    def get_url_params(
+            self,
+            context: dict | None,
+            next_page_token: Any | None,
+    ) -> dict[str, Any]:
+        """Return a dictionary of values to be used in URL parameterization."""
+        params = {"per_page": 1000}
+        if next_page_token:
+            params["cursor"] = next_page_token
+        else:
+            start_time = self.get_start_time(context)
+            params["start_time"] = start_time
+        return params
+
+    def get_start_time(self, context: dict | None) -> int:
+        """Get the start time for the initial incremental export."""
+        replication_key_value = self.get_starting_replication_key_value(context)
+        if replication_key_value:
```

Comment:
The replication key should be always required for incremental, so we don't need to check again.